torch_sim/runners.py

```diff
@@ -490,15 +537,17 @@ class StaticState(type(state)):
         )
 
         props = trajectory_reporter.report(substate, 0, model=model)
-        all_props.extend(props)
+
+        for prop_key in extra_prop_keys:
+            setattr(substate, prop_key, torch.concat([prop[prop_key] for prop in props]))
```

Comment:
I am realizing this is actually going to be a bit tricky, `torch.concat` is only appropriate for atomwise properties while batchwise properties should be stacked. More in review comment.